Source/SwiftLintFramework/Extensions/Configuration+LintableFiles.swift

```diff
@@ -14,6 +14,18 @@ extension Configuration {
         return lintablePaths(inPath: path).flatMap(File.init(path:))
     }
 
+    public func lintableFiles(ofFiles files: [File]) -> [File] {
```

Comment:
`Configuration.lintableFiles(ofFiles:)` is not a very descriptive method name...